wcc-contentful-app/spec/generators/wcc/model_generator_spec.rb

```diff
@@ -7,100 +7,100 @@
 require 'generator_spec'
 require 'timecop'
 
-RSpec.describe Wcc::ModelGenerator, type: :generator do
-  destination Rails.root.join('tmp/generators')
+# RSpec.describe Wcc::ModelGenerator, type: :generator do
```

Comment:
What's the plan for this spec? Right now the entire file is commented out.

Reply:
whoops, my bad